docs/changelog/0.6.1.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: 0.6.1
+---
+
+## pg_analytics
+
+### New Features
+
+- Parquet tables now support foreign keys
+- Normal and materialized views can now be created over parquet tables
```

Comment:
I don't think the bm25 search over parquet tables was added to the previous changelog, we should maybe add it here too

Reply:
I didn't add it because the memory blows up when you try to index a table of meaningful size